Document/0x04i-Testing-user-interaction.md

```diff
@@ -30,6 +32,10 @@ Here is a list of best practices where a user could be informed of:
 - **Specific credentials**: When a user gets a recovery code, a password or a pin from the application (or sets one), instruct the user to never share this with anyone else and that only the app will request it.
 - **Application distribution**: In case of a high-risk application it is recommended to communicate what the official way of distributing the app is. Otherwise, users might use other channels in which they download a compromised version of the application.
 
+#### Access to device-data
+
+Allthough partially covered by the Google Play Store and the Apple App Store, you still need to explain to the user which services your app consumes and why. For instance: does your app require access to the contact list? Explain the user why. Does your app need access to location services of the device? Explain the user why. Does your app use device-identifiers to identify the device? Explain the user why your app needs to do that.
```

Comment:
Maybe add something like: 

Good examples for proper explanations can be found in the: 
- [Apple Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/requesting-permission/ "Apple Human Interface Guidelines") and
- in the [Android App permissions best practices](https://developer.android.com/training/permissions/requesting.html#explain "Android App permissions best practices").